lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3513,10 +3509,20 @@ class PropertyEmitter : public EmitterBase,
 };
 } // end anonymous namespace
 
-void PropertyEmitter::emitProperty(
-    Value property, PropertyPrecedence parenthesizeIfLooserThan) {
+void PropertyEmitter::emitEnabledProperty(
+    Value property, Value enable, PropertyPrecedence parenthesizeIfLooserThan) {
   assert(localTokens.empty());
-  // Wrap to this column.
+
+  // If the property is tied to an enable, emit that.
+  if (enable) {
+    ps << "disable iff" << PP::nbsp << "(~(";
+    ps.scopedBox(PP::ibox2, [&] {
+      emitNestedProperty(enable, PropertyPrecedence::Lowest);
+      ps << "))";
+    });
```

Comment:
I think we can set `Unary` precedence. 
```suggestion
    ps << "disable iff" << PP::nbsp << "(~(";
    ps.scopedBox(PP::ibox2, [&] {
      emitNestedProperty(enable, PropertyPrecedence::Unary);
      ps << ")";
    });
```

Reply:
I forget to mention but could you remove extra parentheses?

Reply:
I added them to properly scope the not, I was running into issues without it